src/elements/emby-slider/emby-slider.js

```diff
@@ -139,8 +139,9 @@ define(['browser', 'dom', 'layoutManager', 'keyboardnavigation', 'css!./emby-sli
             return;
         }
 
+        // TODO: Not sure what this is trying to accomplish.
         if (enableWidthWithTransform == null) {
-            //enableWidthWithTransform = browser.supportsCssAnimation();
+            enableWidthWithTransform = browser.supportsCssAnimation();
```

Comment:
With `transform`, I only see a little color transition.
![slider-1](https://user-images.githubusercontent.com/56478732/87886341-c0c52380-ca24-11ea-8b96-a41ce2d572e1.png)
Without
![slider-2](https://user-images.githubusercontent.com/56478732/87886402-3630f400-ca25-11ea-9752-a95c0c1ba3f6.png)

Reply:
That doesn't look very good, I'll remove all of this as well, then.